clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -2122,7 +2122,19 @@ describe("Consent overlay", () => {
 
         // Toggle the notice, but don't save yet
         cy.getByTestId("toggle-Advertising").click();
-        cy.get("@FidesUIChanged").should("have.been.calledOnce");
+        cy.get("@FidesUIChanged").its("callCount").should("equal", 1);
```

Comment:
@NevilleS Just curious what the difference is here? Just for consistency?

Reply:
Yeah, that's all